CI_JobHistory.py

```diff
@@ -165,9 +165,11 @@ def display_ci_links(config_data):
 
     if JENKINS == "False":
         for ci_name in config_data.keys():
-            j=j+1
-            ci_name_list.append(ci_name)
-            print(j,'',ci_name)
+            if any(w in ci_name for w in job_platform):
```

Comment:
Can you modify this check in such a way that it will be executed only once if job_platform is not passed.